### ISSUE TITLE
appindicator on polybar dissapeared the moment I open a window

### ISSUE THREAD

**Issue by s4if** [hyprwm/Hypr#98]
**Please describe the bug**
The moment I open app window on Hypr, appindicator icons disappeared and can't be clicked.

**Steps to reproduce:**
autostart polybar on hypr.conf (I'm using default polybar config) then open kitty (or other apps)

OS: EndeavourOS
PC: i3 5005U
VGA: Intel Integrated

**Expected behavior**
appindicator icons still exist and can be clicked normally.

**Screenshots**
![gh_attach1](https://user-images.githubusercontent.com/2939779/203216980-fbf07061-e9ec-4e81-b297-12eedc449ff9.gif)

**Comment by vaxerski** [comment]
have you disabled the hypr bar? Other than that I have no idea what would be causing this.

**Comment by s4if** [comment]
Yes, I've disabled hypr bar. I've also turning on and off config stuff that has tray in it but no success found...
Even in my Debian 11 machine it has same exact problem. May be this has the same cause that makes Hypr bar didn't show any icons like in #97 issue...
